packages/client/src/raffles/types/raffles.types.ts

```diff
@@ -0,0 +1,49 @@
+export enum RAFFLE_STATUS {
+  DRAFT = 'Draft',
```

Comment:
Prefer pascal case for enum keys like `Draft` instead of `DRAFT`

Reply:
as below?
```
export enum RaffleStatus {
  Draft = 'Draft',
  Ready = 'Ready',
  AcceptingEntries = 'AcceptingEntries',
  EntriesClosed = 'EntriesClosed',
  WinnersSelection = 'WinnersSelection',
  DrawClosed = 'DrawClosed',
  Close = 'Closed',
}

```

Reply:
Yes, thank you.

Reply:
done